tests/schemas/test_ticket_schema.py

```diff
@@ -13,3 +13,19 @@ def test_valid_payload(self, create_tenant, create_admin_user):
 
         no_validation_errors = {}
         assert no_validation_errors == TicketSchema().validate(valid_payload)
+
+    def test_valid_payload_with_note(
```

Comment:
I don't think this test is necessary. We already have a note schema that has validation tests for a note.